docs/references/cluster-configuration.md

```diff
@@ -98,13 +94,8 @@ gcp:
     zone_suffix: a
 ```
 
-
 Note: the example above shows the defaults.
 
-**Warning 😈:**
-we develop and test with region `us-west2` and have not validated other regions yet.
```

Comment:
maybe let's still keep this as a "we don't regularly test other regions"?

Reply:
Aligning this with the AWS text.